### ISSUE TITLE
cannot clone this repo

### ISSUE THREAD

**Issue by agowa** [jazzband/django-taggit#909]
Hi,

when trying to clone this (and only this) repository using `git clone https://github.com/jazzband/django-taggit.git` I get this error:

```
Cloning into 'django-taggit'...
remote: Enumerating objects: 5165, done.
remote: Counting objects: 100% (1232/1232), done.
remote: Compressing objects: 100% (441/441), done.
error: object c0725b93cd0bfdc1038f45b631273af16a160a6d: zeroPaddedFilemode: contains zero-padded file modes
fatal: fsck error in packed object
fatal: fetch-pack: invalid index-pack output
```

**Comment by rtpg** [comment]
Hey, I don't know what's going on there but searching around it seems like you can turn off the fsck checks? See [this discussion](https://github.com/pallets/flask/issues/2029)

Not clear to me what object this is, though I see the errors when calling `git fsck` locally (well, the warnings....). Maybe a shallow clone (`--depth=1`) might work?

**Comment by agowa** [comment]
I opened this issue to let you know so that you can fix the repository. Even though I can work around the issue doesn't mean everyone can (or should have to). Also it breaks a lot of scripts and I do not know if GUI tools will be able to clone the repo at all...

**Comment by rtpg** [comment]
@agowa of course. Would you know what a fix is? Some of the threads seem to imply I would have to rewrite every SHA (since 2010 in that case), which seems a bit agressive but.... maybe doable given we don't have that much activity on the repo.

**Comment by agowa** [comment]
Sadly that is also the only solution I know of...
